src/3d/symbols/qgspointcloud3dsymbol_p.cpp

```diff
@@ -521,4 +521,118 @@ Qt3DRender::QGeometry *QgsRGBPointCloud3DSymbolHandler::makeGeometry( Qt3DCore::
   return new QgsRGBPointCloud3DGeometry( parent, data, byteStride );
 }
 
+QgsClassificationPointCloud3DSymbolHandler::QgsClassificationPointCloud3DSymbolHandler()
+  : QgsPointCloud3DSymbolHandler()
+{
+
+}
+
+bool QgsClassificationPointCloud3DSymbolHandler::prepare( const QgsPointCloud3DRenderContext &context )
+{
+  Q_UNUSED( context )
+  return true;
+}
+
+void QgsClassificationPointCloud3DSymbolHandler::processNode( QgsPointCloudIndex *pc, const IndexedPointCloudNode &n, const QgsPointCloud3DRenderContext &context )
+{
+  QgsPointCloudAttributeCollection attributes;
+  constexpr int xOffset = 0;
```

Comment:
Did you mean 'const' here? That's what is used for yOffset and zOffset too - good to keep this consistent although there might not be much difference between constexpr and const here...

Reply:
I just copied whatever was in QgsColorRampPointCloud3DSymbolHandler. It seems to be one of the changes @nyalldawson did some days ago. I will change constexpr to const since yOffset and zOffset variables are const